src/util/DefaultValueProcessor.ts

```diff
@@ -0,0 +1,41 @@
+export const addValues = (testCase: any): any => {
+  //create a clone of testCase
+  const resultJson: any = testCase;
+  //.map to return an array of just
+  const nonCoverage: Array<any> = testCase.entry.filter((entry) => {
+    if (entry.resource.resourceType != "Coverage") {
+      return true;
+    }
+  });
+
+  const coverage: Array<any> = [];
+  const foundCoverage: any = testCase.entry.find((entry) => {
```

Comment:
Question for my own info: Would there ever be a case where the `entry` array would have more than one Coverage resource type?
Also I would look into optional chaining and strict equality here.

Reply:
Good call out.. I made a modification to replace find with filter to handle that  and added a unit test to verify